src/combo.ts

```diff
@@ -25,6 +29,58 @@ type BeachTile = { minute: number; row: number; column: number };
 // Unfortunately, this can mean some bad things could happen were our data file to get messed up
 const rareTiles: BeachTile[] = JSON.parse(fileToBuffer("raretiles.json"));
 
+// A list of all the common items found in infreqeunt twinkles to detect when we actually combed an infrequent tile instead of a rare
+const infrequentItemNames: string[] = [
+  "sand dollar",
+  "dull fish scale",
+  "rough fish scale",
+  "lucky rabbitfish fin",
+  "spearfish fishing spear",
+  "Waders",
+  "pristine fish scale",
+  "piece of coral",
+].map((s) => s.toLowerCase());
+
+function getFalseRares(): BeachTile[] {
+  // Get the property and split it by the divider of ;
+  const knownFalseRares = getProperty("combo_false_rares")
+    .split(";")
+    .filter((s) => s.length > 0);
+  const beachTiles: BeachTile[] = knownFalseRares.map((s) => {
+    const [minute, row, col] = s.split(",").map((s) => toInt(s));
+
+    return { minute: minute, row: row, column: col };
+  });
+
+  return beachTiles;
+}
+
+// A list of tiles we know are false rares
+const knownFalseRares = getFalseRares();
```

Comment:
do we want this in this scope?

Reply:
(this is not rhetorical, the answer is allowed to be "sure why not")

Reply:
When you say in scope, do you mean "Do we want to avoid false rares" or "Do we want to expose this const"?

Reply:
Variable scope, not project scope